crates/apollo-mcp-server/src/server/states/running.rs

```diff
@@ -59,19 +59,18 @@ impl Running {
             .cloned()
             .map(|operation| operation.into_inner())
             .filter_map(|operation| {
-                match operation.into_operation(
```

Comment:
Is this related to the fix  or just a style you like more? They look functionally identical I think?